src/azure_openai_sample.py

```diff
@@ -0,0 +1,16 @@
+from openai import AzureOpenAI
+
+client = AzureOpenAI(
+    azure_endpoint="https://<your_APIM_endpoint>.azure-api.net/<your_api_suffix>", #do not add "/openai" at the end here because this will be automatically added by this SDK
+    api_key="<your subscription key>",
```

Comment:
Ah if the code is an actual Python file in the repo, then I definitely recommend os.getenv and then a .env.sample with the environment variables (and .env in .gitignore). We get so many accidental public PRs with internal data, so am trying to reduce leaked key incidents.